Sources/Rosetta/PlayMode/CardSets/BlackTempleCardsGen.cpp

```diff
@@ -2007,6 +2040,15 @@ void BlackTempleCardsGen::AddNeutral(std::map<std::string, CardDef>& cards)
     // RefTag:
     //  - DEATHRATTLE = 1
     // --------------------------------------------------------
+    power.ClearData();
+    power.AddPowerTask(
+        std::make_shared<IncludeTask>(EntityType::MINIONS_NOSOURCE));
+    power.AddPowerTask(std::make_shared<FilterStackTask>(
+        SelfCondList{ std::make_shared<SelfCondition>(
+            SelfCondition::IsMinion()) }));
```

Comment:
Is there a reason why you wrote this code?

Reply:
I added this part to specify "other minions" but I just noticed the line before does that job. I'll delete it.